### ISSUE TITLE
Optimize: bug omitting DC in mix

### ISSUE THREAD

**Issue by Sciss** [Sciss/Negatum#9]
In GitLab by @Sciss on Jun 28, 2020, 13:23

input:
```scala
    NegatumIn()
    val lFDClipNoise    = LFDClipNoise.ar(0.025)
    val min             = lFDClipNoise min 0.64
    val lFDNoise3       = LFDNoise3.ar(2211)
    val lFDNoise1       = LFDNoise1.ar(2538.0)
    val lFPulse         = LFPulse.ar(freq = 337.99133, iphase = 0.0, width = 1.0)
    val mix             = Mix(
      Seq[GE](min, lFDNoise3, 1.0, lFDNoise1, lFPulse))
    NegatumOut(mix)
```
output:
```scala
NegatumIn()
val lFDNoise3 = LFDNoise3.ar(2211.0)
val lFDNoise1 = LFDNoise1.ar(2538.0)
val lFPulse   = LFPulse.ar(freq = 337.99133, iphase = 0.0, width = 1.0)
val dC        = DC.ar(0.64)
val mix       = Mix(Seq[GE](lFDNoise3, lFDNoise1, lFPulse, dC))
NegatumOut(mix)
```
So the DC of 1.0 of the input is silently dropped.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 13:36

mentioned in commit be5f26a815e0c7c023452add65a26e5bf8a4ec8d

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 14:38

What I don't understand is how constant can end up in the `Mix` at all. Looking at `MkSynthGraph` I can't seem to find where this could happen.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 14:58

It happens because of the optimisation in `Vertex.UGen.Impl.mkBinOpUGen` and `mkUnOpUGen`. This can result in constants.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 18:51

mentioned in commit 151a222a36065eb5b48714b16d18ade808fcad71

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 22:31

closed via commit 151a222a36065eb5b48714b16d18ade808fcad71

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 22:31

closed via commit be5f26a815e0c7c023452add65a26e5bf8a4ec8d
